### ISSUE TITLE
My playgound for the same requirements...

### ISSUE THREAD

**Issue by WernerMairl** [karelz/GitHubIssues#171]
Hi
This project seems to be sleeping ;-)

Inspired by your work here i have created my own try/prototype
focused on a smaller implementation  in pure dotnet core.

https://github.com/WernerMairl/GGQL

any feedback is welcome!
What is the plan here ?

regards
Werner

**Comment by karelz** [comment]
Yes, the project is a bit sleeping :( ... I was able to get back to the new design 2-3 weeks ago and have an early prototype of consuming [GH events](https://developer.github.com/v3/activity/events/types/) (via internal MS channel over ServiceBus) with ~350K events for last few weeks to test on. That would enable "real-time" events instead of regular snapshots.
I also prototyped some code to represent and process the notifications. However, it needs more flashing out.

Interestingly it seems that even the GH events are missing some "interesting" notifications which show up only in history - links between issues, issue/comment reactions, etc. While it is not a big deal, I am toying with the idea to use "history crawler" as alternative source of events - at least for initializing the "database".

Timeframe wise: I hope I will be able to get back to work on the v2 tooling in late November and during December.

Let me know if you want to chat on gitter or somewhere about ideas, lessons learned, etc.
